addon/lib/document.js

```diff
@@ -127,6 +127,21 @@ export default Base.extend({
       console.debug(`${label}:`, any);
       return any;
     });
+    documentJexl.addTransform("min", (arr) => Math.min(...arr));
+    documentJexl.addTransform("max", (arr) => Math.max(...arr));
+    documentJexl.addTransform("round", (num, places) =>
+      isNaN(num)
+        ? null
+        : !isNaN(places)
```

Comment:
Why? This isn't user input. We can assume that this is a number

Reply:
That is true, fixed